save-core/src/commonMain/kotlin/org/cqfn/save/core/Save.kt

```diff
@@ -169,7 +172,7 @@ class Save(
         }
     }
 
-    @Suppress("WHEN_WITHOUT_ELSE")  // TestResult is a sealed class
+    @Suppress("WHEN_WITHOUT_ELSE")  // TestStatus is a sealed class
```

Comment:
why?

Reply:
TestResult is not a sealed class )
TestStatus is